satpy/tests/reader_tests/conftest.py

```diff
@@ -15,4 +15,201 @@
 #
 # You should have received a copy of the GNU General Public License along with
 # satpy.  If not, see <http://www.gnu.org/licenses/>.
+
 """Setup and configuration for all reader tests."""
+
+import datetime as dt
+import os
+from random import randrange
+
+import numpy as np
+import pytest
+import xarray as xr
+from trollsift import compose, parse
+from xarray import DataTree
+
+from satpy.readers.mwr_l1b import AWS_EPS_Sterna_MWR_L1BFile
+from satpy.readers.mwr_l1c import AWS_MWR_L1CFile
+
+DATETIME_FORMAT = "%Y-%m-%d %H:%M:%S.%f"
+
+platform_name = "AWS1"
+# W_XX-EUMETSAT-Darmstadt,SAT,AWS1-MWR-1B-RAD_C_EUMT_20241121085911_G_D_20241109234502_20241110004559_T_N____.nc
+file_pattern = "W_{country:2s}-{organisation:s}-{location:s},SAT,{platform_name}-MWR-{processing_level}-RAD_C_{originator:4s}_{processing_time:%Y%m%d%H%M%S}_G_D_{start_time:%Y%m%d%H%M%S}_{end_time:%Y%m%d%H%M%S}_T_B____.nc"  # noqa
+
+
+rng = np.random.default_rng()
+
+def random_date(start, end):
+    """Create a random datetime between two datetimes."""
+    delta = end - start
+    int_delta = (delta.days * 24 * 60 * 60) + delta.seconds
+    random_second = randrange(int_delta)
+    return start + dt.timedelta(seconds=random_second)
+
+
+@pytest.fixture(scope="session")
```

Comment:
Should this also be module scoped like the other fixtures?

Reply:
Yes, I think so. Changed. Thanks.